tabs/src/components/app/views/QueryRunner.jsx

```diff
@@ -43,6 +43,7 @@ export function QueryRunner(props) {
         setIsLoading(true);
         const queryParameters = query.substring(GRAPH_URL.length + graphVersion.length, query.length);
         const graphResponse = await makeGraphCall(requestType, requestHeaders, queryParameters, graphVersion, requestBody, locale);
+        setIsLoading(false);
```

Comment:
what happens if the promise is unresolved? eg. if there's an error
does the setIsLoading(false) still execute?

Reply:
resolved offline :)

Reply:
3e8a0fe55e6e230e105be8a8bf9a15273ae29a4b Handled it in a similar way to the Graph Explorer web app